mangopy/mango.py

```diff
@@ -0,0 +1,187 @@
+# mango.py
+# Basic data and plotting utilities for MANGO network
+#
+# created 2019-3-19 by LLamarche
+# Notes:
+# - Default data directory is TEMP/MANGOData/, where TEMP is defined by tempfile.gettempdir() (https://docs.python.org/3/library/tempfile.html)
+# - TODO: Data files availabe at ftp://isr.sri.com/pub/earthcube/provider/asti/MANGOProcessed/
+
+import numpy as np
+import datetime as dt
+import h5py
+import csv
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import os
+import urllib
+from contextlib import closing
+import tempfile
+
+
+class Mango(object):
+
+    def __init__(self, datadir=None):
+
+        self.mangopy_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        # if no data directory specified, use a default temp directory
+        if datadir is None:
+            datadir = os.path.join(tempfile.gettempdir(),'MANGOData')
```

Comment:
Maybe one should be verbose about the creation of this directory? Print something to let the user know?

Reply:
I'm fine printing a statement to tell users what has been chosen as the default data directory.  Do you think it makes more sens to do it here or down at line 109 where the directory is actually created?

Reply:
Whoops, ya, definitely makes more sense to do it where the directory is created.